server/controllers/finance/cash.js

```diff
@@ -273,7 +273,7 @@ function reference(req, res, next) {
       FROM cash JOIN project ON cash.project_id = project.id
     )c WHERE c.reference = ?;`;
 
-  db.one(sql, [ ref ])
+  db.one(sql, [ ref , 'cash'])
```

Comment:
Typo - nice catch.  It should read `db.one(sql, [ref], ref, 'cash');`.  That way it will print out a nice error message if this fails.

Although .. I'm actually not sure this route is ever used.